CHANGELOG.md

```diff
@@ -3,6 +3,87 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 7.6.3
+
+_Jun 13, 2024_
+
+We'd like to offer a big thanks to the 12 contributors who made this release possible. Here are some highlights ✨:
+
+- 🎁 Allow customization of the Pickers month and the year buttons
```

Comment:
Is there anything else you see worth highlighting?

Reply:
Shouldn't it be a minor since there is a feature? :angel:

Reply:
Fair point. 🤔 
This and the Data grid change warrants a minor. 👌